python/python/opm/io/deck/__init__.py

```diff
@@ -1 +1,32 @@
 from opm._common import DeckKeyword
+from opm._common import DeckItem
+
+
+def defaulted_deckitem(self):
+    return self.__defaulted(0)
+
+
+def has_value_deckitem(self):
+    return self.__has_value(0)
+
+
+def get_item_deckitem(self):
+
+    if self.is_int():
+        return self.get_int(0)
+    elif self.is_string():
+        return self.get_str(0)
+    elif self.is_double():
+        return self.get_raw(0)
+    elif self.is_uda():
+        if self.__is_numberic():
+            return self.__uda_double()
+        else:
+            return self.__uda_str()
+    else:
+        raise ValueError("Deck Item, unknown type")
+
+
+setattr(DeckItem, "defaulted", defaulted_deckitem)
```

Comment:
Personally I like to use properties in Python - where it makes sense - here you could make `defaulted`, `has_value`, `get` into properties (I have not tested this ....):
```Python
@property
det get_item_deckitem(self)
      ....

setattr(DeckItem, "value", get_item_deckitem)
```
and similar for the other two; would maybe also suggest the property name `valid` for `has_value_deckitem()`?

Reply:
I have not used the property decorator before, but google is very helpful and I now see what you are suggesting with the comment below. However, this will not work with the suggestion above (to use a optional parameter, SI=True in the get function).  I'm happy to add the property decorator if we agree to drop the additional SI argument in the get function

Reply:
@joakim-hove, I have now followed up all your comments . Thanks a lot for prioritizing this. I know that you have a lot to do these days. The @property decorator was new to me, very useful I would say.